src/api/middleware/usersMiddleware.ts

```diff
@@ -39,6 +39,19 @@ export const isUserBelongsToParticipant = async (
   return false;
 };
 
+const canUserAccessParticipant = async (
+  requestingUserEmail: string,
+  participantId: number,
+  traceId: string
+) => {
+  const isRequestingUserUid2Support = await isUid2Support(requestingUserEmail);
+
+  return (
+    isRequestingUserUid2Support ||
+    (await isUserBelongsToParticipant(requestingUserEmail, participantId, traceId))
+  );
+};
```

Comment:
super nit: just a thought, fine with what you decide - it seemed a little weird to put the result of `isUid2Support` into a variable, but not do the same for `isUserBelongsToParticipant` - I would say it could be more consistent, either both variables or both calling the function in the return statement

Reply:
good point - I created this variable before splitting out the `canUserAccessParticipant` function - it's probably not necessary now. I'm putting them both in the return statement so we can benefit from an early return if the user is UID2 support